policy-controller/src/admission.rs

```diff
@@ -57,23 +88,28 @@ pub fn routes(client: kube::Client) -> BoxedFilter<(impl warp::Reply,)> {
                 }
             };
             debug!(?rsp);
-            ok(reply::json(&rsp.into_review()))
+            json_response(rsp.into_review())
         })
-        .with(warp::trace::request())
-        .boxed()
+    }
 }
 
-type Review = kube::core::admission::AdmissionReview<DynamicObject>;
-type Request = kube::core::admission::AdmissionRequest<DynamicObject>;
-type Response = kube::core::admission::AdmissionResponse;
-
-#[inline]
-fn ok(reply: impl warp::Reply + 'static) -> Result<Box<dyn warp::Reply>, Infallible> {
-    Ok(Box::new(reply))
+fn json_response(rsp: AdmissionReview) -> Result<Response<Body>> {
+    let bytes = serde_json::to_vec(&rsp)
+        .map_err(|e| anyhow::Error::from(e).context("failed to encode admission review as JSON"))?;
```

Comment:
I mentioned this in the comments of my other PR but this might be a little cleaner:

```suggestion
    use anyhow::Context;
    
    let bytes = serde_json::to_vec(&rsp)
        .context("failed to encode admission review as JSON")?;
```